components/heroes/SkillSection.tsx

```diff
@@ -12,16 +12,15 @@ export function SkillSection({
   return (
     <div className="grid grid-cols-12 items-center mt-2 mb-2">
       <div className="col-span-12 text-center sm:col-span-1">
-        <SkillIcon skill={skill}></SkillIcon>
+        <SkillIcon skill={skill} />
       </div>
       <div className="col-span-12 text-center sm:col-span-11 sm:text-left">
         <p className="text-2xl">
           {isAwakening && "Awakening: "}
           {skill.name}
         </p>
         <p>
-          Cost: {skill.cost} / CD:{skill.cd} / Range:{skill.range} / Span:
-          {skill.span}
+          {`Cost: ${skill.cost} / CD: ${skill.cd} / Range: ${skill.range} / Span: ${skill.span}`}
```

Comment:
Is there a reason this is a preferred syntax?

Reply:
no special reason, i think. Just a habit i guess. Or perhaps i didnt like {" "} after Span In my defend its more clear to see "spaces". But i agree. should write in same style. Fixed.